src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/InstanceNameTest/InstanceNameTest.cs

```diff
@@ -31,6 +32,17 @@ public static void ConnectToSQLWithInstanceNameTest()
                 using SqlConnection connection = new(builder.ConnectionString);
                 connection.Open();
                 connection.Close();
+
+                if (builder.Encrypt != SqlConnectionEncryptOption.Strict)
+                {
+                    // Exercise the IP address-specific code in SSRP
+                    IPAddress[] addresses = Dns.GetHostAddresses(hostname);
+                    builder.DataSource = builder.DataSource.Replace(hostname, addresses[0].ToString());
+                    builder.TrustServerCertificate = true;
+                    using SqlConnection connection2 = new(builder.ConnectionString);
+                    connection2.Open();
+                    connection2.Close();
+                }
```

Comment:
Removing the 5.0 preq for `Encrypt` is still an `boolean` and not a `SqlConnectionEncryptOption` enum in 4.0
```suggestion
                // Exercise the IP address-specific code in SSRP
                IPAddress[] addresses = Dns.GetHostAddresses(hostname);
                builder.DataSource = builder.DataSource.Replace(hostname, addresses[0].ToString());
                builder.TrustServerCertificate = true;
                using SqlConnection connection2 = new(builder.ConnectionString);
                connection2.Open();
                connection2.Close();
```